modules/ROOT/pages/provisioning-raspberry-pi4.adoc

```diff
@@ -45,6 +45,8 @@ sudo dnf install -y --downloadonly --release=$RELEASE --forcearch=aarch64 --dest
 
 Now extract the contents of the RPMs and copy the proper `u-boot.bin` for the RPi4 into place:
 
+WARNING: The following commands to extract the contents of the RPMs, the use of `coreos-installer`, and copying of the files to the ESP partition should be done from outside of a Toolbx container.
```

Comment:
I wonder if we should add some explanation of why. I think the issue you were hitting was because `root` inside the toolbx container was not the same as `UID=0` on the host.

Reply:
Sure, I can expand on this.